gematria/datasets/pipelines/compile_modules_lib.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2024 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import apache_beam as beam
+from rules_python.python.runfiles import runfiles
+import os
+
+from absl import logging
+
+from collections.abc import Iterable
+from collections.abc import Callable
+
+import subprocess
+
+from gematria.datasets.python import extract_bbs_from_obj
```

Comment:
The imports should be sorted:
1. standard library imports (`collections.abc`, `os`, `subprocess`),
2. third-party library imports (`absl`, `apache_beam`, `rules_python`),
3. project imports (anything under `gematria`).

See https://google.github.io/styleguide/pyguide.html#313-imports-formatting for the detailed rule.